build/scripts/build.ps1

```diff
@@ -86,7 +86,8 @@ function Run-MSBuild([string]$buildArgs = "", [string]$logFile = "") {
     }
 
     if ($cibuild) { 
-        $args += " /p:PathMap=`"$($repoDir)=q:\roslyn`" /p:Feature=pdb-path-determinism" 
+        $pathMapDir = "q:\roslyn"
```

Comment:
I can't bring myself to put anything directly into the root of a drive. Even if it's just lying about putting it there.